docs/diginstroom/sip/1.0/sip_structure/5_structure_package.md

```diff
@@ -1818,6 +1821,22 @@ TODO: figure out the IDs
 | Cardinality | 1..1 |
 | Obligation | MUST |
 
+| Element | `premis:premis/premis:event/premis:eventOutcomeInformation` |
+|-----------------------|-----------|
+| Name | Event outcome information  |
+| Description | Information about the outcome of an event. |
+| Cardinality | 0..1 |
```

Comment:
In de KG is dit momenteel verplicht, maar wellicht moet dit niet. Ik paste het aan.

Reply:
Kan ook naar SHOULD gezet worden. Heb er zelf niet zo'n mening over.